package.json

```diff
@@ -6,25 +6,16 @@
 		"url": "https://github.com/Automattic/newspack-plugin/issues"
 	},
 	"homepage": "https://newspack.com",
-	"engines": {
-		"node": "16",
-		"npm": "8"
-	},
 	"devDependencies": {
-		"@rushstack/eslint-patch": "^1.10.2",
+		"@rushstack/eslint-patch": "^1.10.3",
 		"@testing-library/react": "^12.1.4",
 		"@types/qs": "^6.9.15",
 		"@types/react": "^17.0.75",
-		"@types/wordpress__block-editor": "^11.5.14",
-		"@types/wordpress__blocks": "^12.5.14",
-		"@types/wordpress__components": "^23.0.12",
-		"@wordpress/browserslist-config": "^5.40.0",
-		"eslint": "^7.32.0",
+		"@wordpress/browserslist-config": "^6.0.0",
+		"eslint": "^8.57.0",
 		"lint-staged": "^15.2.2",
-		"newspack-scripts": "^5.3.0",
-		"postcss-scss": "^4.0.9",
-		"prettier": "npm:wp-prettier@^2.6.2-beta-1",
```

Comment:
I'm not sure how Prettier and ESLint should be interacting. I found that when I kept the Prettier dependency, it was linting with very different style/formatting rules than ESLint. When using `wp-scripts lint-js` it seems to have built-in formatting rules (I think it already invokes Prettier under the hood), so I'm not sure a separate Prettier dependency or config is still necessary?

Reply:
With `v5.6.0-alpha.*` versions of newspack-scripts, `lint:js` and `format:js` scripts seem to be working with WP formatting standards, and it's also formatting correctly in my VSCode.

Reply:
In my testing, running `npm run format:js` reformats JS, introducing changes to most of the files. Running `prettier` directly (`./node_modules/.bin/prettier src --write`) has the same effect (though it will also format SCSS).

Reply:
Yes, `@wordpress/scripts` enforces different formatting rules than what we've been using as our in-house style (which is closer to ESLint's formatting, e.g. with spaces in brackets like `( foo, bar )` and `[ a, b ]`. So this is technically working, but I'm not sure we want to change our team's coding style to conform to this just yet, which is why I [removed Prettier as a requirement to pass the `lint` command](https://github.com/Automattic/newspack-scripts/pull/209/files#diff-1bd145b27e5bc507fcffdc88b445a1b341284a8f581d53375452838316832a41R75) in our own ESLint config.

What's nice about `@wordpress/scripts` is that linting and formatting can be treated separately. So if `npm run lint:js` throws autofixable errors, we should be able to run `wp-scripts lint-js --fix` to autofix those without enforcing Prettier formatting styles. Would it be helpful in the short term to replace the `format:js` commands with a `fix:js` command that runs this, and then reintroduce the Prettier-based `format:js` script in the future if we decide to follow the Prettier rules from `@wordpress/scripts`?

We could

Reply:
c580196 temporarily removes the `format:js` script and introduces a `fix:js` script which basically runs `eslint --fix` via @wordpress/scripts. This should cover autofixable code quality errors without addressing the conflicting Prettier formatting rules at this time.

Reply:
Oh! In the flurry I missed your initial mention of `"prettier": "npm:wp-prettier@^2.6.2-beta-1",`. Restoring the wp-prettier dependency in [dcd4d33](https://github.com/Automattic/newspack-scripts/pull/209/commits/dcd4d33ee7c803f99332ae5273202e313be9a916) does mostly fix the formatting conflicts! So I restored the `format:js` script in ebf4197a. However, running this still results in a lot of (but fewer) changes to JS src files, so I won't include a wholesale reformatting of JS in this changeset.